assets/src/modules/Action.js

```diff
@@ -56,8 +56,17 @@ export default class Action {
 
     /**
      * Build the lizmap Action instance
+     * @param {Map}           map           - OpenLayers map
+     * @param {SelectionTool} selectionTool - The lizmap selection tool
+     * @param {object}        lizmap3       - The old lizmap object
+     * @param {Digitizing}    digitizing    - The Lizmap digitizing instance
      */
-    constructor() {
+    constructor(map, selectionTool, lizmap3, digitizing) {
```

Comment:
It is hoped that the use of lizmap3 will disappear and be replaced by the new API. Can you set lizmap3 as the last parameter?